broadway_api/callbacks/worker.py

```diff
@@ -34,14 +34,14 @@ def _handle_lost_worker_node(settings, worker):
 
     if not lost_run_id:
         logger.critical(
-            "worker '{}' ({}) went offline unexpectedly".format(
-                worker.hostname, worker.id
+            "worker {} went offline unexpectedly on {}".format(
+                worker.id, worker.hostname
             )
         )
         return
     logger.critical(
```

Comment:
Nit: Nothing you added but I think this log was supposed to be in the `else` branch

Reply:
Like this?
```
 if not lost_run_id:
        logger.critical(
            "worker {} went offline unexpectedly on {}".format(
                worker.id, worker.hostname
            )
        )
        return
else:
    logger.critical(
        "worker {} went offline unexpectedly on {} while executing '{}'".format(
            worker.id, worker.hostname, lost_run_id
        )
    )
```

Reply:
Yes I think that's what was intended